modules/administration-guide/partials/con_backup-and-recovery.adoc

```diff
@@ -0,0 +1,14 @@
+[id="backup-and-recovery_{context}"]
+= Backup and recovery
+
+Backing up {prod-short} involves a combination of the following processes that back up respective data:
+
+* Backing up and restoring the {prod-short} instance by using {prod-short} Operator and a xref:supported-restic-compatible-backup-servers.adoc[configured backup server]. You can back up the {prod-short} instance using custom resources or {prod-cli} to a backup server that you have configured: xref:backups-of-che-instances-to-an-sftp-backup-server.adoc[SFTP], xref:backups-of-che-instances-to-amazon-s3.adoc[Amazon S3] (or S3 API compatible storage), or xref:backups-of-che-instances-to-a-rest-backup-server.adoc[REST]. You can restore the {prod-short} instance using xref:restoring-a-che-instance-from-a-backup.adoc#restoring-a-che-instance-from-a-backup-using-the-checlusterrestore-object_{context}[custom resources] or xref:restoring-a-che-instance-from-a-backup.adoc#restoring-a-che-instance-from-a-backup-using-prod-cli_{context}[{prod-cli}].
++
+NOTE: For testing this process, {prod-short} offers the xref:backups-of-che-instances-to-the-internal-backup-server.adoc[internal backup server].
+
+* Backing up and restoring the source code stored in users' workspaces by using xref:backups-of-persistent-volumes.adoc[backups of persistent volumes].
++
+TIP: Whether or not backups of persistent volumes are implemented, users can be advised to commit and push their changes to avoid losing their work.
```

Comment:
```suggestion
TIP: Whether or not you implement backups of persistent volumes, advise users to commit and push their changes to avoid losing their work.
```
- Avoid passive voice. Be authoritative.
- Is this the correct formatting for a Tip admonitiion?

Reply:
Thanks. Fixed.